vNext/src/OrleansAzureUtils/project.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "supports": {},
+  "targets": {
+    ".NETStandard,Version=v1.5": {
+      "Microsoft.Data.OData/5.6.4": {
+        "compile": {
+          "lib/portable-net45+wp8+win8+wpa/Microsoft.Data.OData.dll": {}
+        },
+        "runtime": {
+          "lib/portable-net45+wp8+win8+wpa/Microsoft.Data.OData.dll": {}
+        }
+      },
+      "Microsoft.Data.Services.Client/5.6.4": {
+        "compile": {
+          "lib/portable-net45+wp8+win8+wpa/Microsoft.Data.Services.Client.dll": {}
+        },
+        "runtime": {
+          "lib/portable-net45+wp8+win8+wpa/Microsoft.Data.Services.Client.dll": {}
+        }
+      },
+      "System.Spatial/5.6.4": {
+        "compile": {
+          "lib/portable-net45+wp8+win8+wpa/System.Spatial.dll": {}
+        },
+        "runtime": {
+          "lib/portable-net45+wp8+win8+wpa/System.Spatial.dll": {}
+        }
+      }
+    }
+  },
+  "dependencies": {
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+    "NETStandard.Library": "1.6.0",
```

Comment:
(not an issue) We typically take a "latest version - 1" approach to dependencies. For .NET Standard, I believe we should adopt the latest version of every package before our initial release, because it's all moving so rapidly.